src/components/Calculator.js

```diff
@@ -1,54 +1,44 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-    this.handler = this.handler.bind(this);
-  }
+function Calculator() {
```

Comment:
- [ ] [OPTIONAL] Kindly change the function syntax to es6, so it will be like 
```
const Calculator = () => {
}
```

Reply:
done :heavy_check_mark: